kaiax/gov/param.go

```diff
@@ -51,11 +53,45 @@ var (
 		}
 
 		switch v := v.(type) {
-		case []byte:
-			return stringToAddressList(string(v))
-		case string:
+		case []byte: // input from header.Vote
+			// There are three types of header.Vote encoding for validator address(es).
+			// Type1. Single address, [20]byte. See Mainnet block 5505383.
+			// Type2. Multiple addresses, [20*n]byte.
+			// Type3. Single address, [42]byte (hex-encoded bytes). See Mainnet block 90915008.
```

Comment:
Are you going to make it valid? It could have been a bug in governance_vote

Reply:
I think it was a bug. Where do you think is appropriate if not here?

Reply:
We could ignore such vote I think? currently you are parsing the data without problem.

Reply:
Without type3 parsing logic here, `VerifyHeader()` will fail during sync test.
With this PR, `NewVoteData()` should return a canonicalized vote data on validator votes. (Previously, it returned `voteData` without value)

Reply:
I see. Then it accepts hex-encoded address as valid vote, but will not produce anymore via governance_vote.